problem5/calculate.c

```diff
@@ -36,6 +36,12 @@ int total(int numbers[], int length) {
 
 
 bool areStringsEqual(const char* first, const char* second) {
+
+    int strcmp(const char* first, const char* second);
+    int result;
+
+    return result;
```

Comment:
Finally, in C, we have to make sure that the function returns a value that has the same type specified in the function declaration. So, notice that the function is declared as follows:

```c
bool areStringsEqual(const char* first, const char* second) {
```

Notice that in front of the function name (`areStringsEqual`) we have a `bool` type. This means that the function should return a `bool` (true/false) value. Here, you're returning `result`, which is an `int` value. That means, we'll have to convert `result` from an `int` to a `bool`. If you need help figuring this one out, check out this page: https://www.tutorialspoint.com/c_standard_library/c_function_strcmp.htm